precompiles/avs/avs.sol

```diff
@@ -20,7 +20,8 @@ interface IAVSManager {
         string[] memory assetID,
         uint64 action,
         uint64 minSelfDelegation,
-        uint64 unbondingPeriod
+        uint64 unbondingPeriod,
```

Comment:
For my understanding, is there any reason why `uint64` was chosen? Not a blocker for this PR.

Reply:
There's no special reason for this